plugins/all-modules-page/src/test/kotlin/org/jetbrains/dokka/allModulesPage/templates/AddToNavigationCommandResolutionTest.kt

```diff
@@ -0,0 +1,137 @@
+package org.jetbrains.dokka.allModulesPage.templates
+
+import kotlinx.html.a
+import kotlinx.html.div
+import kotlinx.html.id
+import kotlinx.html.span
+import kotlinx.html.stream.createHTML
+import org.jetbrains.dokka.DokkaModuleDescriptionImpl
+import org.jetbrains.dokka.allModulesPage.MultiModuleAbstractTest
+import org.jetbrains.dokka.base.renderers.html.templateCommand
+import org.jetbrains.dokka.base.templating.AddToNavigationCommand
+import org.jetbrains.dokka.plugability.DokkaContext
+import org.junit.Rule
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+import org.junit.rules.TemporaryFolder
+
+class AddToNavigationCommandResolutionTest : MultiModuleAbstractTest() {
+    @get:Rule
+    val folder: TemporaryFolder = TemporaryFolder()
```

Comment:
IMHO this folder could be extracted to `MultiModuleAbstractTest` in all those tests

Reply:
No, because this way you force every test to use a temporary folders, which is not needed if we want to test eg. AllModulesPageGeneration on some mocked data.

Reply:
Then we could have a mixin for that, not worth the effort though